src/nvim/api/window.c

```diff
@@ -470,3 +471,29 @@ void nvim_win_config(Window window, Integer width, Integer height,
     win->w_pos_changed = true;
   }
 }
+
+/// Close a window.
+///
+/// This is equivalent to |:close| with count except that it takes a window id.
+///
+/// @param window   Window handle
+/// @param force    Behave like `:close!` The last window of a buffer with
+///                 unwritten changes can be closed. The buffer will become
+///                 hidden, even if 'hidden' is not set.
+
+/// @param[out] err Error details, if any
```

Comment:
I think the docstring needs to be "connected" for doxygen to map it. so line 483 should have `///`